frontend/src/concepts/connectionTypes/types.ts

```diff
@@ -87,13 +87,14 @@ export type ConnectionTypeField =
 export type ConnectionTypeConfigMap = K8sResourceCommon & {
   metadata: {
     name: string;
-    annotations: DisplayNameAnnotations & {
+    annotations?: DisplayNameAnnotations & {
       'opendatahub.io/enabled'?: 'true' | 'false';
       'opendatahub.io/username'?: string;
     };
-    labels: DashboardLabels & {
+    labels?: DashboardLabels & {
       'opendatahub.io/connection-type': 'true';
     };
+    ownerReferences?: OwnerReference[];
```

Comment:
Doesn't `K8sResourceCommon` already give us `ownerRefernces` prop? I think this is redundant.